mmseg/models/backbones/erfnet.py

```diff
@@ -0,0 +1,326 @@
+        output = self.bn(output)
+        output = self.act(output)
+        return output
+
+
+class NonBottleneck1d(BaseModule):
+    """Non-bottleneck block of ERFNet.
+
+    Args:
+        channels (int): Number of channels in Non-bottleneck block.
+        drop_rate (float): Probability of an element to be zeroed.
+            Default 0.
+        dilation (int): Dilation rate for last two conv layers.
+            Default 1.
+        conv_cfg (dict | None): Config of conv layers.
+            Default: None.
+        norm_cfg (dict | None): Config of norm layers.
+            Default: dict(type='BN').
+        act_cfg (dict): Config of activation layers.
+            Default: dict(type='ReLU').
+        init_cfg (dict or list[dict], optional): Initialization config dict.
+            Default: None.
+    """
+
+    def __init__(self,
+                 channels,
+                 drop_rate=0,
+                 dilation=1,
+                 conv_cfg=None,
+                 norm_cfg=dict(type='BN', eps=1e-3),
+                 act_cfg=dict(type='ReLU'),
+                 init_cfg=None):
+        super(NonBottleneck1d, self).__init__(init_cfg=init_cfg)
+
+        self.conv_cfg = conv_cfg
+        self.norm_cfg = norm_cfg
+        self.act_cfg = act_cfg
+        self.act = build_activation_layer(self.act_cfg)
+
+        self.convs_layer = nn.ModuleList()
+        for conv_layer in range(2):
+            conv_first_padding = (1, 0) if conv_layer == 0 else (1 * dilation,
+                                                                 0)
+            conv_first_dilation = 1 if conv_layer == 0 else (dilation, 1)
+            conv_second_padding = (0, 1) if conv_layer == 0 else (0, dilation)
+            conv_second_dilation = 1 if conv_layer == 0 else (1, dilation)
+
+            self.convs_layer.append(
+                build_conv_layer(
+                    self.conv_cfg,
+                    channels,
+                    channels,
+                    kernel_size=(3, 1),
+                    stride=1,
+                    padding=conv_first_padding,
+                    bias=True,
+                    dilation=conv_first_dilation))
+            self.convs_layer.append(self.act)
+            self.convs_layer.append(
+                build_conv_layer(
+                    self.conv_cfg,
+                    channels,
+                    channels,
+                    kernel_size=(1, 3),
+                    stride=1,
+                    padding=conv_second_padding,
+                    bias=True,
+                    dilation=conv_second_dilation))
+            self.convs_layer.append(
+                build_norm_layer(self.norm_cfg, channels)[1])
+            if conv_layer == 0:
+                self.convs_layer.append(self.act)
+            else:
+                self.convs_layer.append(nn.Dropout(p=drop_rate))
+
+    def forward(self, input):
+        output = input
+        for op in self.convs_layer:
```

Comment:
```suggestion
        for conv in self.convs_layer:
```

Reply:
Not only convolution, but also `build_activation_layer` and `build_norm_layer`, I do not want to change because `op` operations is more reasonable.